lib/RSet.cpp

```diff
@@ -51,7 +51,7 @@ void RObject::setRSet(RSet* rs, int rcnt) {
    rcount = rcnt;
 #ifdef DEBUG
    if (rcount > references)
-      printd(0, "RObject::setRSet() this: %p '%s' cannot set rcount %d > references %d\n", this, getName(), rcount, references);
+      printd(0, "RObject::setRSet() this: %p '%s' cannot set rcount %d > references %d\n", this, getName(), rcount, (int)references);
```

Comment:
I would use `references.load()` here to be consistent with the rest of the code. Same with the comparisons above and under this line.